include/sys/euiAnimator.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <nn/font/font_Util.h>
+
+#include <nn/ui2d/ui2d_Animation.h>
+#include <nn/ui2d/ui2d_Group.h>
+#include <nn/ui2d/ui2d_Pane.h>
+
+#include <eui/lytex/euiLayoutEx.h>
+
+namespace eui {
+
+class Animator : public AnimTransformBasic {
+public:
+    NN_RUNTIME_TYPEINFO_BASE();

```

Comment:
This is probably not right, I would check the functions in the binary and see if they reference the parents' runtime vars (in which case use the other macro)

Reply:
I've changed the implementation of this to now use NN_RUNTIME_TYPEINFO with AnimTransformBasic. Should be approximate to what the original headers used.